qt/python/mantidqtinterfaces/mantidqtinterfaces/Muon/GUI/Common/pairing_table_widget/pairing_table_widget_view.py

```diff
@@ -203,32 +252,36 @@ def _guess_alpha_button(self):
     def guess_alpha_clicked_from_row(self):
         self._on_guess_alpha_clicked(self.pairing_table.currentIndex().row())
 
-    def get_table_contents(self):
-        if self._updating:
-            return []
-        ret = [[None for _ in range(self.num_cols())] for _ in range(self.num_rows())]
-        for row in range(self.num_rows()):
-            for col in range(self.num_cols()):
-                if pair_columns[col] == "group_1" or pair_columns[col] == "group_2":
-                    # columns with widgets
-                    ret[row][col] = str(self.pairing_table.cellWidget(row, col).currentText())
-                elif pair_columns[col] == "guess_alpha":
-                    ret[row][col] = "Guess"
-                else:
-                    # columns without widgets
-                    ret[row][col] = str(self.pairing_table.item(row, col).text())
-        return ret
-
     def get_table_item(self, row, col):
         return self.pairing_table.item(row, col)
 
-    def get_table_item_text(self, row, col):
-        if pair_columns[col] == "group_1" or pair_columns[col] == "group_2":
-            return str(self.pairing_table.cellWidget(row, col).currentText())
-        elif pair_columns[col] == "guess_alpha":
-            return "Guess"
-        else:  # columns without widgets
-            return str(self.pairing_table.item(row, col).text())
+    def set_widget_enabled(self, row, col, enabled):
+        cell_widget = self.pairing_table.cellWidget(row, col)
+        cell_widget.setEnabled(enabled)
+
+    def set_item_selectable(self, row, col):
+        item = self.get_table_item(row, col)
+        item.setFlags(QtCore.Qt.ItemIsSelectable)
+
+    def set_item_selectable_and_enabled(self, row, col):
+        item = self.get_table_item(row, col)
+        item.setFlags(self.get_default_item_flags())
```

Comment:
This line is giving an error on runtime as you are trying to access a property..
```suggestion
        item.setFlags(self.get_default_item_flags)
```